code/modules/paperwork/paper_premade.dm

```diff
@@ -1,76 +1,178 @@
-
-/obj/item/paper/fluff/jobs/security/court_judgement
-	name = "paper- 'Judgement'"
-	info = "For crimes against the station, the offender is sentenced to:<BR>\n<BR>\n"
-
-/obj/item/paper/fluff/jobs/toxins/chemical_info
-	name = "paper- 'Chemical Information'"
-	info = "Known Onboard Toxins:<BR>\n\tGrade A Semi-Liquid Plasma:<BR>\n\t\tHighly poisonous. You cannot sustain concentrations above 15 units.<BR>\n\t\tA gas mask fails to filter plasma after 50 units.<BR>\n\t\tWill attempt to diffuse like a gas.<BR>\n\t\tFiltered by scrubbers.<BR>\n\t\tThere is a bottled version which is very different<BR>\n\t\t\tfrom the version found in canisters!<BR>\n<BR>\n\t\tWARNING: Highly Flammable. Keep away from heat sources<BR>\n\t\texcept in an enclosed fire area!<BR>\n\t\tWARNING: It is a crime to use this without authorization.<BR>\nKnown Onboard Anti-Toxin:<BR>\n\tAnti-Toxin Type 01P: Works against Grade A Plasma.<BR>\n\t\tBest if injected directly into bloodstream.<BR>\n\t\tA full injection is in every regular Med-Kit.<BR>\n\t\tSpecial toxin Kits hold around 7.<BR>\n<BR>\nKnown Onboard Chemicals (other):<BR>\n\tRejuvenation T#001:<BR>\n\t\tEven 1 unit injected directly into the bloodstream<BR>\n\t\t\twill cure unconscious and sleep toxins.<BR>\n\t\tIf administered to a dying patient it will prevent<BR>\n\t\t\tfurther damage for about units*3 seconds.<BR>\n\t\t\tit will not cure them or allow them to be cured.<BR>\n\t\tIt can be administered to a non-dying patient<BR>\n\t\t\tbut the chemicals disappear just as fast.<BR>\n\tMorphine T#054:<BR>\n\t\t5 units will induce precisely 1 minute of sleep.<BR>\n\t\t\tThe effect are cumulative.<BR>\n\t\tWARNING: It is a crime to use this without authorization"
-
-
-	/*
-	*	Stations
-	*/
-
-
-///////////	CentCom
-
-/obj/item/paper/fluff/stations/centcom/disk_memo
-	name = "memo"
-	info = "GET DAT FUKKEN DISK"
-
-/obj/item/paper/fluff/stations/centcom/broken_evac
-	info = "Due to circumstances beyond our control, your Emergency Evacuation Shuttle is out of service.<br><br>We apologize for the inconvenience this may cause you.<br><br>Please enjoy the use of this complementary book.<br><br>Sincerely,<br>CentCom Operations Demolitions Examination Retribution Bugfixing Underlining Services"
-
-/obj/item/paper/fluff/stations/centcom/bulletin
-	name = "paper- 'Official Bulletin'"
-	info = "<BR>CentCom Security<BR>Port Division<BR>Official Bulletin<BR><BR>Inspector,<BR>There is an emergency shuttle arriving today.<BR><BR>Approval is restricted to Nanotrasen employees only. Deny all other entrants.<BR><BR>CentCom Port Commissioner"
-
-
-///////////	Lavaland
-
-/obj/item/paper/fluff/stations/lavaland/orm_notice
-	name = "URGENT!"
-	info = "A hastily written note has been scribbled here... <br><br> Please use the ore redemption machine in the cargo office for smelting. PLEASE! <br><br>--The Research Staff"
-
+/*

```

Comment:
Damn line endings, this is why I tell you to use git instead of uploading your own files.

Reply:
I'm sorry about being captain shitcode but it was all I was able to do quickly. If this doesn't work, I'll have to revisit it later today.

Reply:
Should be sorted now.